### ISSUE TITLE
Gisportal: Deleting projects ist not deleting projectfiles on server

### ISSUE THREAD

**Issue by mrkskrnr** [uprel/gisapp#98]
The function for admin to edit projects on gisportal includes the possibility of deleting a project. So doing this, project disappears on gisportal, but .qgs-file still remains on server in file directory. It would be great to delete also the .qgs-file and related files (.html and .json) on server.

**Comment by uprel** [comment]
True, this only removes project from database. Will see about deleting on disk also
